docs/api/structures/custom-scheme.md

```diff
@@ -1,7 +1,7 @@
 # CustomScheme Object
 
 * `scheme` String - Custom schemes to be registered with options.
-* `options` Object (optional)
+* `privileges` Object (optional)
```

Comment:
Unless I'm misreading this (which is possible), this looks like an unadvertised breaking API change. This goes against our [policy](https://github.com/electron/electron/blob/master/docs/api/breaking-changes.md#api-contract) of announcing these changes at least one major version ahead of time.

If this is targeted for 6.0, please make sure to update `breaking-changes.md` in `5-0-x` before merging this.